dist/tools/doccheck/generic_exclude_patterns

```diff
@@ -17,3 +17,6 @@ warning: Member LED[0-9]_PIN_NUM \(macro definition\) of
 warning: Member LED[0-9]_PORT \(macro definition\) of
 warning: Member LED[0-9]_PORT_NUM \(macro definition\) of
 warning: Member LED[0-9]_TOGGLE \(macro definition\) of
+warning: Member SHT1X_PARAMS \(macro definition\) of
+warning: Member SHT1X_PARAM_[A-Z0-9_]* \(macro definition\) of
+warning: Member SHT1X_SAULINFO \(macro definition\) of
```

Comment:
this does not seem generic to me each macro is defined exactly once all in the same file

Reply:
All FOO_PARAM_<val> are generic. IMO those should be added with the driver, so that future PRs adding boards do not need to worry about the doccheck anymore.

Reply:
the added the missing generic documentation has these never hit for our codebase  (the warning does not happen why filter for it) 
```suggestion
```